src/DotNetWorker.Core/WorkerHostedService.cs

```diff
@@ -14,17 +14,17 @@ internal class WorkerHostedService : IHostedService
         private readonly IWorker _worker;
         private readonly IWorkerDiagnostics _diagnostics;
 
-        public WorkerHostedService(IWorker worker, IWorkerDiagnostics diagnostics)
+        public WorkerHostedService(IWorker worker)
         {
             _worker = worker ?? throw new ArgumentNullException(nameof(worker));
-            _diagnostics = diagnostics ?? throw new ArgumentNullException(nameof(diagnostics));
+            //_diagnostics = diagnostics ?? throw new ArgumentNullException(nameof(diagnostics));
```

Comment:
@brettsam will remove this and replace with an issue link to address the logging at initialization